<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainpage</class>
 <widget class="QMainWindow" name="mainpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>677</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>mainpage</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">mainpage {
background-color: rgb(241, 242, 246); 
}

Line {
	color: 1px solid red;
}

.QPushButton {
background-color: rgb(193, 193, 193);

border-radius: 7px;
padding-left: 15px;
text-align: left;
}

.QPushButton::hover {
background-color: rgb(241, 242, 246); 
}

.QPushButton::click {
background-color: rgb(238, 238, 238); 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>263</width>
      <height>523</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="leftpage">
     <item>
      <layout class="QVBoxLayout" name="operation">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="settingText">
           <property name="minimumSize">
            <size>
             <width>251</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>251</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:gray;</string>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="datasetBtt">
             <property name="minimumSize">
              <size>
               <width>251</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>251</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>üìÇ Dataset</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchbyBtt">
             <property name="minimumSize">
              <size>
               <width>251</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>251</width>
               <height>31</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>üóù Search by</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="extensionText">
               <property name="minimumSize">
                <size>
                 <width>251</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>251</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color:gray;</string>
               </property>
               <property name="text">
                <string>Extensions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="favlistBtt">
               <property name="minimumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>üíå Favorite list</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="historyBtt">
               <property name="minimumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>üìù History</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="quizBtt">
               <property name="minimumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>251</width>
                 <height>31</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>‚ùì Quiz</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="topMargin">
                <number>100</number>
               </property>
               <item>
                <widget class="QLabel" name="groupText">
                 <property name="styleSheet">
                  <string notr="true">color:gray;</string>
                 </property>
                 <property name="text">
                  <string>¬© Group 6</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="courseText">
                 <property name="minimumSize">
                  <size>
                   <width>251</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>251</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:gray;</string>
                 </property>
                 <property name="text">
                  <string>CS163 APCS 2021-2022 </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="recommendationBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#recommendationBar{

border-radius: 7px;
padding-left: 5px;
padding-right: 5px;
background-color: rgb(226, 226, 232);
}

#recommendationBar::item:selected {
	border-radius: 4px;
	background-color: rgb(204, 0, 30);
	color:white;
     border-style: outset;
}

#recommendationBar::item:hover {
	border-radius: 4px;
	background-color:  rgb(91, 174, 255);
	color:white;
     border-style: outset;
}

#recommendationBar::item {
	margin-top: 5px;
	height: 23px;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>253</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <widget class="QLineEdit" name="searchBar">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#searchBar {
border-radius: 7px;
padding-left: 5px;
background-color: rgb(226, 226, 232);
transition: ease;
}

#searchBar::focus{
border: 2px solid rgb(120, 159, 240);
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string comment="üîé Searching here..." extracomment="üîé Searching here...">üîé Searching here...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>16</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">Line {
	stroke: red;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>511</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalWidget_8" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="word_2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>39</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>39</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>WORD</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>39</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButton_16">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>‚ô°</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_17">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>‚úé</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_18">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>‚å¶</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="def_2">
         <property name="text">
          <string>This is the area of the definition of the above word. You might have some 
trouble with entering.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>491</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_19">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_20">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_21">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Change mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_22">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>461</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Favorite List</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actiondjkfjskdf">
   <property name="text">
    <string>djkfjskdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>click()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
